mitosheet/src/mito/components/toolbar/DataTabContents.tsx

```diff
@@ -18,31 +17,6 @@ export const DataTabContents = (
     }): JSX.Element => {
 
     return (<div className='mito-toolbar-bottom'>
-        <ToolbarButton
-            action={props.actions.buildTimeActions[ActionEnum.Import_Files]}
-        />
-        <ToolbarButton
-            action={props.actions.buildTimeActions[ActionEnum.Dataframe_Import]}
-        />
-        <ToolbarButton
-            action={props.actions.buildTimeActions[ActionEnum.SNOWFLAKEIMPORT]}
-        />
-        {props.actions.runtimeImportActionsList.map(action => {
-            return (<ToolbarButton
-                action={action}
-                iconOverride={<ImportIcon />}
-                key={action.staticType}
-            />)
-        })}
-
-        <div className='toolbar-vertical-line' />
-        
-        <ToolbarButton
-            action={props.actions.buildTimeActions[ActionEnum.UPDATEIMPORTS]}
```

Comment:
In Excel, these are in the Data tab. I think we should break from that based on results testing, rather than just blind intuition.

That being said, we can keep this for now.